### DISCUSSION TITLE
RadioReg Release

### DISCUSSION THREAD

**Discussion by wegGehamstert** [AzuraCast/AzuraCast#7333] [Show and tell]
@Optischa and me released a project at the 14 July Falles **RadioReg**

**What exactly should RadioReg be?**
RadioReg should be a directory for web radios. Of course, this already exists, especially on the web. Well-known platforms for this include Radio.de or onlineradiobox.com.

However, we want to take the whole thing to a new level. We started with a Labymod Addon (a modification for Minecraft wich allows the players to listen to music ingame) and launched also a website called https://RadioReg.net

**What's the idea behind this?**
The BreathFM team wants to revive the remaining web radio scene and no longer see the others as competition. We would support each other with this project because we offer a platform with which music can be listened to for free and also offers smaller radio stations the opportunity to be discovered.

We think it's time to do something. This means we can finally unite all radios (at least in part) and offer our listeners maximum comfort. This also means that smaller web radios can be found more quickly.

**How can you help us?**
We would be very happy if you would join our project and register your radio. In addition, we also need people who can program and implement these ideas.
That's why we created a Discord server where we can exchange ideas and continue pursuing this vision.
You can join at: http://dc.radioreg.net

![Radio_Reg-removebg-preview](https://github.com/user-attachments/assets/db5774b6-0ca5-484c-87cc-089bfbe0f0d4)

**Comment by BornKillaz** [comment]
Hi! Congratulations on your project and idea.

But, how exactly can we register a station?

Thanks.

**Comment by punyauk** [comment]
I managed to sign up and add my station but notice the tags are missing both 'Community' and also 'LGBT'

**Reply by wegGehamstert** [reply]
We will consider adding those tags in the near future. If you find some more that are missing, just write unter my comment

**Reply by mdrxy** [reply]
`college` would be appreciated as well

**Comment by BornKillaz** [comment]
I also manage to register a station, but is not showing the now playing data. @wegGehamstert, is that normal?

**Reply by wegGehamstert** [reply]
It's normal if you didn't connected a Webhook trough azuracast 

But thanks for testing out out new service. The overall project is at a early stage, so some bugs are still needed to get fixed

**Reply by BornKillaz** [reply]
Ah, didn't even knew that there was already a RadioReg webhook defined in Azuracast. So, it's done and metadata is being sent. Station is BassTik Radio. Thanks. 👍 💯

**Comment by gualtierogrey** [comment]
Hi, created organization but when creating Radio it shows an internal server error

**Reply by wegGehamstert** [reply]
Hey, what's your organisation called?

**Reply by gualtierogrey** [reply]
Ldn house radio
